### PR TITLE
Buffs Psydrain.

### PR DESCRIPTION
**Pull request by Sebbe9123**
## About The Pull Request
This PR buffs psydrain and gives it an auxiliary purpose to make dead marines dead-er.

## Why It's Good For The Game
Makes marine deaths less free by making even front lines deaths more impactful and debilitating unless marines can get you out INSTANTLY. Clone damage can be fixed with the crunch tube (cryo tube) and a battle-nap.

## Changelog
:cl:
add: Ads 20 CLONE damage to psydrain.
/:cl:

### CODE REVIEW

**Comment by jroinc** [comment]
I would personally recommend cloneloss, as brainloss will just result in the ungas putting alkysine in all their shit. 

Not a bad idea, though.

**Comment by BraveMole** [comment]
> I would personally recommend cloneloss, as brainloss will just result in the ungas putting alkysine in all their shit.
> 
> Not a bad idea, though.

Isn't cloneloss easy to heal as well with resting though?

**Comment by Sebbe9123** [comment]
> I would personally recommend cloneloss, as brainloss will just result in the ungas putting alkysine in all their shit.
> 
> Not a bad idea, though.

Changed to be clone damage instead of brain damage.

**Comment by jroinc** [comment]
> 
> 
> > I would personally recommend cloneloss, as brainloss will just result in the ungas putting alkysine in all their shit.
> > Not a bad idea, though.
> 
> Isn't cloneloss easy to heal as well with resting though?

Yes, but there's a difference.

If you take 20 brainloss, all the robust ungas and medics will instantly swap to 5u alkysine injectors or pill bottles, and it just becomes *another* game of "give the corpse another pill for a fullheal", or "now that I'm revived I'll use my custom med to instaheal it." and it'll be gone before you can stand up. 

And if you *don't* have that, it'll cripple you with random stuns, so it preys on newbies while not doing anything to the old zoomers.

If you take 20 cloneloss, you can't fix that with a med or anything other than a deliberate decision to NOT fight for a bit. It'll take... around one to two minutes on a bed with sheets to heal that off, and about that amount of time in medevac to drop pod with cryo, so it's an existing consequence.

If you choose to ignore it, that's fine. It's only 20 damage. It won't stun you or disable your arms or whatever horror story other shit does. But it's still there as a weakness. Start getting psydrained over and over again, and you'll get very frail till you can be shipsided.

File: code/modules/mob/living/carbon/xenomorph/abilities.dm [hunk 1]
```diff
@@ -1207,6 +1207,11 @@
 
 	victim.do_jitter_animation(2)
 
+	if(ishuman(victim))
+		var/mob/living/carbon/human/H = victim
+		victim.emote_burstscream()
+		H.adjustCloneLoss(20)
```
Comment (ivanmixo) [inline code/modules/mob/living/carbon/xenomorph/abilities.dm:1213]: Why

```suggestion
		victim.adjustCloneLoss(20)
```

Comment (Sebbe9123) [reply code/modules/mob/living/carbon/xenomorph/abilities.dm:1213]: Idk tbh, I originally had it as brain damage so I just used what worked. I'll let an actual maintainer commit this if it's better or not.

Comment (BraveMole) [reply code/modules/mob/living/carbon/xenomorph/abilities.dm:1213]: 1) remove the emote, the victim is dead, it's stupid
2) you cannot psydrain non human, so the ischeck is useless
3) cloneloss is a living proc, so casting to human is useless

**Review by BraveMole — CHANGES_REQUESTED** [review]

**Comment by Sebbe9123** [comment]
Think it's compliant now. Scream was because we sucking their psionic energy, felt kinda fitting.

**Review by BraveMole — APPROVED** [review]

**Pull request merged**